examples/binding-scala/js/src/main/scala/com/thoughtworks/todo/Main.scala

```diff
@@ -0,0 +1,148 @@
+package com.thoughtworks.todo
+
+import com.thoughtworks.binding.Binding._
+import com.thoughtworks.binding.dom
+import scala.collection.GenSeq
+import scala.scalajs.js.annotation.JSExport
+import org.scalajs.dom._
+import org.scalajs.dom.ext._
+import org.scalajs.dom.raw._
+import upickle.default._
+
+@JSExport object Main {
+
+  final class Todo(val title: String, val completed: Boolean)
+
+  object Todo {
```

Comment:
Right. I solved that by generating ids for the todos. Both approaches work, but as it stands i would have left a comment